includes/Jobs/ClearGitInfoCache.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Miraheze\MirahezeMagic\Jobs;
+
+use Job;
+use MediaWiki\MainConfigNames;
+use MediaWiki\MediaWikiServices;
+
+class ClearGitInfoCache extends Job {
+
+	public const JOB_NAME = 'ClearGitInfoCache';
+
+	private array $keys;
+	private string $startWiki;
+
+	public function __construct( array $params ) {
+		parent::__construct( self::JOB_NAME, $params );
+
+		$this->startWiki = $params['startWiki'];
+		$this->keys = $params['keys'];
+	}
+
+	public function run(): bool {
+		$cache = MediaWikiServices::getInstance()->getObjectCacheFactory()->getInstance( CACHE_ANYTHING );
+
+		foreach ( $this->getConfig()->get( MainConfigNames::LocalDatabases ) as $db ) {
+			foreach ( $this->keys as $key ) {
+				$startWiki = $this->startWiki;
```

Comment:
Use preg_quote() just in case, and move it outside the loop

Reply:
I've never used preg_quote before, how exactly would I use it here?

Reply:
Oh I can just do preg_quote( $this->startWiki )?

Reply:
Don't forget to add "/" as the second parameter (see docs for why)

Reply:
Thanks, done.